google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/Partition.java

```diff
@@ -96,6 +96,10 @@ ByteString getPartitionToken() {
     return partitionToken;
   }
 
+  public ByteString getPartitionTokenForTest() {
```

Comment:
can this be package private?

Reply:
We intend to make it public to get them out of this class for test. Actually that's what we want to do.

Reply:
We should be super careful to not have test classes / methods in the public GA API.  You can create a test utility in the `com.google.cloud.spanner` project, so that we don't have a public test method in the class we expose.

Reply:
What do you mean by creating a test utility? Basically we want something that can get the partitionToken variable out of the partition class.

Reply:
I mean creating a `com/google/cloud/spanner/PartitionTestUtil.java` class that would be able to access the package-private variables, and implementing `PartitionTestUtil.extractPartitionToken(partition)` which will call `Partition.getPartitionToken()`

Reply:
Sure, will do that.